### ISSUE TITLE
"Congratulations: Next question" popup shown after getting a question wrong

### ISSUE THREAD

**Issue by jsharkey13** [ucam-cl-dtg/isaac-app#653]
https://github.com/ucam-cl-dtg/isaac-app/commit/2b0e25eeac58b5a745362604d615b355e24111a0
#419
#584 

You can get a question incorrect and get the following response:
![image](https://cloud.githubusercontent.com/assets/8060558/12854045/5ea5dd2c-cc2f-11e5-8fbd-4e01d27ca271.png)

It appears to be a bug introduced by the commit above, related to the two issues above.
I don't think it's specific to SymbolicQuestions, but there's some code that is - so it might be.

**Comment by allydavies** [comment]
I have seen a similar (you have achieved mastery) message under these circumstances:

I have previously acheived mastery, and went to reenter an answer. I entered a wrong answer to demonstrate the useful feedback message related to that question; and was congratulated for having achieved mastery.

I remember previously asking for the mastery achieved message to come up ONLY once when mastery was first achieved. I thought that had been implemented too.
